core/iwasm/interpreter/wasm_loader.c

```diff
@@ -10888,6 +10887,12 @@ wasm_loader_get_custom_section(WASMModule *module, const char *name,
 }
 #endif
 
+#if 0
+#define HANDLE_OPCODE(opcode) #opcode
+DEFINE_GOTO_TABLE(const char *, op_mnemonics);
```

Comment:
Is this mnemonics opcode table aimed to facilitate debugging in wasm_loader?

Reply:
Yes, in wasm_loader_prepare_bytecode, when traversing opcodes, we can print opcode name, e.g. `printf("##%s\n", op_mnemonics[opcode]);`.

Reply:
thanks, got it